webpack.config.js

```diff
@@ -10,7 +10,6 @@ module.exports = {
   },
   output: {
     path: path.resolve(__dirname, "sphinx_wagtail_theme/static/dist"),
-    publicPath: "",
```

Comment:
**Question:** why did you remove this option? Will removing it have adverse effects?

Reply:
Sorry. I should have called this out. The upgrade for css extract recommended removing this as it's no longer needed. 

Webpack will default this to `auto` instead. 

https://webpack.js.org/guides/public-path/
https://github.com/webpack-contrib/mini-css-extract-plugin/blob/master/CHANGELOG.md#200-2021-06-30

Reply:
Thanks! Sounds fine to me.